src/framework/main_control_loop.py

```diff
@@ -31,7 +33,10 @@ def setup_control_tasks(self):
 
     def execute(self):
         """Call execute on all control tasks in order."""
+        start_time = time.time()
+        self.sfr.set("start_time",start_time)
```

Comment:
can we instead use an "end of last control cycle time that is set at the end of execute in clock_manager?

Reply:
we want to keep MCL clean of any lower-level manipulations of the SFR to maintain modularity.